Author: Kurt Cancemi <kurt@x64architecture.com>
Description: Fix build error which is caused by removed OpenSSL unused error codes
Forwarded: yes
--- a/vendor/nginx-1.11.1/src/event/ngx_event_openssl.c
+++ b/vendor/nginx-1.11.1/src/event/ngx_event_openssl.c
@@ -1999,7 +1999,6 @@
             || n == SSL_R_ERROR_IN_RECEIVED_CIPHER_LIST              /*  151 */
             || n == SSL_R_EXCESSIVE_MESSAGE_SIZE                     /*  152 */
             || n == SSL_R_LENGTH_MISMATCH                            /*  159 */
-            || n == SSL_R_NO_CIPHERS_PASSED                          /*  182 */
             || n == SSL_R_NO_CIPHERS_SPECIFIED                       /*  183 */
             || n == SSL_R_NO_COMPRESSION_SPECIFIED                   /*  187 */
             || n == SSL_R_NO_SHARED_CIPHER                           /*  193 */
